frontend/tests/unit/calculators/census-tracts-test.js

```diff
@@ -15,18 +16,28 @@ module('Unit | Calculator | census-tracts', function(hooks) {
       'railroad',
     ];
 
-    const autoCensusTract1 = 165;
-    const autoCensusTract2 = 108;
-    const taxiCensusTract1 = 237;
-    const taxiCensusTract2 = 113;
-    const busCensusTract1 = 0;
-    const busCensusTract2 = 0;
-    const subwayCensusTract1 = 1642;
-    const subwayCensusTract2 = 800;
-    const walkCensusTract1 = 605;
-    const walkCensusTract2 = 458;
-    const railCensusTract1 = 77;
-    const railCensusTract2 = 9;
+    this.autoCensusTract1 = 165;
+    this.autoCensusTract2 = 108;
+    this.taxiCensusTract1 = 237;
+    this.taxiCensusTract2 = 113;
+    this.busCensusTract1 = 0;
+    this.busCensusTract2 = 0;
+    this.subwayCensusTract1 = 1642;
+    this.subwayCensusTract2 = 800;
+    this.railCensusTract1 = 77;
+    this.railCensusTract2 = 9;
+    this.walkCensusTract1 = 605;
+    this.walkCensusTract2 = 458;
+    this.ferryCensusTract1 = 0;
+    this.ferryCensusTract2 = 0;
+    this.streetcarCensusTract1 = 0;
+    this.streetcarCensusTract2 = 0;
+    this.bicycleCensusTract1 = 82;
+    this.bicycleCensusTract2 = 50;
+    this.motorcycleCensusTract1 = 0;
+    this.motorcycleCensusTract2 = 0;
+    this.otherCensusTract1 = 17;
+    this.otherCensusTract2 = 33;
 
     const censusTractVariables = [
```

Comment:
Here's how I would organize this so it's easier to follow:

```javascript
// put this in the outer scope:
function createCensusTractDummyData(geoid) {  
  return {
    population: {
      moe: 569,
      geoid,
      value: 5415,
      variable: 'population',
    },
    trans_home: {
      moe: 153,
      geoid,
      value: 385,
      variable: 'trans_home',
    },
     trans_taxi: {
      moe: 154,
      geoid,
      value: 237, // NOTE: Use hard-coded value
      variable: 'trans_taxi',
    },
    // others...
  }
};

```

Then, in each of your tests:

```javascript
const censusTractVariables = [createCensusTractDummyData(1234), createCensusTractDummyData(5678)];
```

Reply:
ohhh I see, using another way to shorten the code to the human eye, but to the machine it is still "hardcoding".  Whereas previously I was shortening to the machine, but leaving it verbose to the human eye.

Reply:
I might not be able to use the helper, because the different census tracts have different mode values. I may just have to duplicate censusTractsVariable array across tests.

Reply:
that's okay! make two helpers if you think it'll be better. or just duplicate :)